app/controllers/posts_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
-    @comment = Comment.new
+    @comments = @post.comments.includes([:author])
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure the app solves the N+1 problem when fetching all posts and their comments for a user as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLAZ4K2ZHA777536NECDBHORQS)
- You could also use this approach: `@posts = @user.posts.includes(:comments)`  instead of @posts = @user.posts. For more documentation, visits the [Solving the N+1 problem in Rails](https://medium.com/idopterlabs/solving-the-n-1-problem-in-rails-b5ea677e78e0)